_data/security.yml

```diff
@@ -40,6 +40,13 @@ websites:
       hardware: Yes
       exceptions:
           text: "Requires X.509-capable cryptotoken or smartcard."
+    
+    - name: Cloudflare
+      url: https://cloudflare.com
+      img: cloudflare.png
```

Comment:
The `cloudflare.png` image needs to be moved to the `/img/security/` folder.

Reply:
@jamcat22 how do I do this via Github.com? I tried cloning the repo and doing it there, but I get a 403 when trying to push. Any ideas?